traittypes/traittypes.py

```diff
@@ -19,6 +19,10 @@ def __getattribute__(self, name):
     import pandas as pd
 except ImportError:
     pd = _DelayedImportError('pandas')
+try:
+    import xarray as xr
+except ImportError:
+    xr = _DelayedImportError('xarray')
```

Comment:
This (`xr`) is actually not used anywhere, so you can remove these lines.

Reply:
Right, and `pd` also (for pandas).